metacat-functional-tests/metacat-test-cluster/etc-metacat/catalog/embedded-fast-hive-metastore.properties

```diff
@@ -0,0 +1,78 @@
+metacat.thrift.port=12004
+connector.name=hive
+hive.use.embedded.metastore=true
+hive.use.embedded.fastservice=true
+hive.exec.compress.intermediate=true
+hive.exec.max.dynamic.partitions.pernode=10000
+hive.exec.max.dynamic.partitions=300000
+hive.exec.parallel=true
+hive.exec.reducers.max=200
+hive.exec.scratchdir=/tmp/metacat/scratch
+hive.hadoop.supports.splittable.combineinputformat=true
+hive.intermediate.compression.codec=org.apache.hadoop.io.compress.SnappyCodec
+hive.intermediate.compression.type=BLOCK
+hive.mapjoin.localtask.max.memory.usage=0.9
+hive.mapjoin.smalltable.filesize=100000000
+hive.mapred.mode=strict
+hive.optimize.s3.query=true
+hive.ppd.remove.duplicatefilters=false
+hive.security.authorization.createtable.owner.grants=ALL
+hive.security.authorization.enabled=true
+mapred.max.split.size=200000000
+webinterface.private.actions=true
+hive.metastore.warehouse.dir=file://tmp/hive/warehouse
+hive.metastore.checkForDefaultDb=true
+
+datanucleus.autoCreateSchema=true
+datanucleus.fixedDatastore=false
+datanucleus.autoCreateTables=true
+
+fs.s3.impl=org.apache.hadoop.fs.s3.S3FileSystem
+fs.s3n.impl=org.apache.hadoop.fs.s3native.NativeS3FileSystem
+fs.default.name=hdfs://10.155.133.15:9000
+hadoop.tmp.dir=/tmp/metacat/hadoop/tmp
+fs.s3.buffer.dir=/tmp/metacat/hadoop/s3,/tmp/metacat/hadoop/s3
+io.compression.codecs=org.apache.hadoop.io.compress.GzipCodec,org.apache.hadoop.io.compress.DefaultCodec,com.hadoop.compression.lzo.LzoCodec,com.hadoop.compression.lzo.LzopCodec,org.apache.hadoop.io.compress.BZip2Codec,org.apache.hadoop.io.compress.SnappyCodec
+fs.s3bfs.impl=org.apache.hadoop.fs.s3.S3FileSystem
+hadoop.metrics.defaultFile=/home/hadoop/conf/hadoopDefaultMetricsList
+hadoop.metrics.list=TotalLoad,CapacityTotalGB,UnderReplicatedBlocks,CapacityRemainingGB,PendingDeletionBlocks,PendingReplicationBlocks,CorruptBlocks,CapacityUsedGB,numLiveDataNodes,numDeadDataNodes,MissingBlocks
+io.compression.codec.lzo.class=com.hadoop.compression.lzo.LzoCodec
+ipc.client.connect.max.retries=50
+fs.s3n.endpoint=s3-external-1.amazonaws.com
+s3mper.disable=false
```

Comment:
Are you using s3mper in the tests?

Reply:
Configurations from the past. We will have to revisit this.